pkg/api/v1alpha1/hostosconfig_types.go

```diff
@@ -29,4 +29,17 @@ type BottlerocketConfiguration struct {
 
 	// Boot defines the boot settings for bottlerocket.
 	Boot *v1beta1.BottlerocketBootSettings `json:"boot,omitempty"`
+
+	// CertBundles defines the additional trusted certbundles for bottlerocket.
+	// +optional
+	CertBundles []certBundle `json:"certBundles,omitempty"`
```

Comment:
I would place it under `HostOSConfiguration` directly in case we need to add this for other providers in the future.
For now, we can add a preflight that checks this field is specified only if the osFamily is set to br